src/Symfony/Component/Yaml/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+ * Add new `lint:yaml dirname --exclude=/dirname/foo.yml --exclude=/dirname/bar.yml`
+   option to exclude one or more specific files from multiple file list
+ * Allow negatable for the parse tags option with `--no-parse-tags` and increase
+   the `symfony/console` dependency to version 5.3
```

Comment:
Not sure we need to mention the increase

Reply:
Indeed

Reply:
👍🏻  removed.

Reply:
👍🏻  I leave it as it is.